Utils/Hash/openssl/sha.h

```diff
@@ -7,7 +7,7 @@
  * The implementation was written so as to conform with Netscape's SSL.
  *
  * This library is free for commercial and non-commercial use as long as
- * the following conditions are aheared to.  The following conditions
+ * the following conditions are adhered to.  The following conditions
```

Comment:
this license block is a bit of a mess since it's original to the license block. I have an experimental feature that lets me ignore blocks instead of lines, but I'm not particularly comfortable with the design.

As I just made a PR yesterday to a project which unfortunately had included `wil` in `expect.txt`, I'd encourage people use `patterns.txt` or `excludes.txt` if they really want to accept specific instances of typos, lest they accidentally enable typos to spread.

#### `patterns.txt`
```
# openssl license typo
the following conditions are adhered to
```

#### `excludes.txt`
```
^\QUtils/Hash/openssl/sha.h\E$
```